.github/workflows/ci.yml

```diff
@@ -157,19 +156,15 @@ jobs:
 
       - name: Build debug
         if: matrix.kind == 'test_debug'
-        run: |
-          echo ::set-env name=DENO_BUILD_MODE::debug
-          cargo build --locked --all-targets
+        run: cargo build --locked --all-targets
 
       - name: Test release
         if: matrix.kind == 'test_release'
         run: cargo test --release --locked --all-targets
 
       - name: Test debug
         if: matrix.kind == 'test_debug'
-        run: |
-          echo ::set-env name=DENO_BUILD_MODE::debug
-          cargo test --locked --all-targets
+        run: cargo test --locked --all-targets
 
       - name: Run Benchmarks
         if: matrix.kind == 'bench'
```

Comment:
I would suggest to change `python ./tools/benchmark.py target/release` to `python ./tools/benchmark.py --release` and remove any further arguments parsing logic from `benchmark.py`.